packages/mobile/App.svelte

```diff
@@ -122,6 +120,9 @@
         await getVersionDetails()
         await pollMarketData()
         await pollNetworkStatus()
+
+        const initialLocale = $appRoute === AppRoute.Welcome ? null : $appSettings.language
```

Comment:
I think still we need a way to detect the device language, maybe we could use [device.getlanguagecode()](https://capacitorjs.com/docs/apis/device#getlanguagecode)
With the capacitor v4 we also could use `device#getlanguageTag` to process also the region code and possibly apply to the amount input to handle the `,.` issue

Reply:
> When tested with a clean install I still getting English as default, having Spanish defined on my device, maybe using the capacitor device plugin could help us to fix it

Yeah, seems there is still an issue. It works on my iOS device but not on may Android it seems, just checked again. Do you have the problem on both OS?

Reply:
To reproduce on iOS we need to add a new language and keep English or any other there should be more than 1, then uninstall the app, change the language and install again. The JS way to get the language returns the first language of the list, but not the selected one on the OS. With Capacitor device plugin we get the correct selected language

Reply:
> I think adding only the [device.getlanguagecode()](https://capacitorjs.com/docs/apis/device#getlanguagecode) would be enough, forget by now the Capacitor v4 update, I've been testing on this branch some time ago but we'll need to check some issues https://github.com/iotaledger/firefly/tree/chore/update-capacitor-v4

Alright, I added the getLanguageCode function and it seems to work know 👍 Could you check again? 😄

Reply:
now works! but seems there is still some issue with 'es' and 'pt' lang codes, I'll fix it